BuildResidentialHPXML/measure.rb

```diff
@@ -1802,18 +1802,18 @@ def arguments(model)
     arg.setDefaultValue(0)
     args << arg
 
-    arg = OpenStudio::Measure::OSArgument::makeDoubleArgument('plug_loads_television_annual_kwh', true)
+    arg = OpenStudio::Measure::OSArgument::makeDStringArgument('plug_loads_television_annual_kwh', true)
     arg.setDisplayName('Plug Loads: Television Annual kWh')
     arg.setDescription('The annual energy consumption of the television plug loads.')
     arg.setUnits('kWh/yr')
-    arg.setDefaultValue(620)
+    arg.setDefaultValue(Constants.Auto)
     args << arg
 
-    arg = OpenStudio::Measure::OSArgument::makeDoubleArgument('plug_loads_other_annual_kwh', true)
+    arg = OpenStudio::Measure::OSArgument::makeStringArgument('plug_loads_other_annual_kwh', true)
```

Comment:
Note that when you make changes like this, the osw files will no longer run via `openstudio run -w foo.osw` because it complains that the osws have arguments with the wrong type (double instead of string). Not sure how important it is, but I try to remember to update tasks.rb for changes like this. The osw's do run via the meta-measure because it does the datatype conversions automatically, so that's why our tests still work.

Reply:
You're saying that in tasks.rb, the setArgument argument needs to be a string, and not a double anymore?

Reply:
Yes. Technically.

Reply:
Ok, I will make those changes.